website/docs/content/writing-style/index.md

```diff
@@ -0,0 +1,306 @@
+
+When labeling buttons, use the format [Verb + Noun] wherever possible. For example:  Add cluster, etc. “Add” and “Create” are frequently used. The general guidelines for their use are as follows:
+
+- **Create/Delete**: Used when a new entity is produced or destroyed in the system. Example create/delete a project (the project didn’t exist before/will no longer exist).
+- **Add/Remove**: Used when a new relationship or dependency between existing objects is produced or destroyed. Example: Add/Remove a user to/from a project (both the user and the project already existed and will continue to exist, but the relationship itself is established or dissolved).
+
+!!! Do
+
+“Create cluster” or “Delete cluster”
+
+Add user to organization
+
+!!!
+
+!!! Dont
+
+“Create cluster” on a button and “Deploy cluster” on the resulting dialog
+
+Using “remove” and “delete” or “create” and “add” interchangeably
+
+!!!
+
+For actions within an object, use the icon (when possible), the verb and the noun that will be affected.
+
+!!! Do
+
+“Edit project” or “View Project”
+
+![A menu with two actions in it with icons](/assets/content/writing-style/writing-style-consistent-termonology-menu-actions-do.png)
+
+!!!
+
+!!! Dont
+
+Miss the object that the user is doing the action. View what?
+
+![A menu with two actions in it, with no icons](/assets/content/writing-style/writing-style-consistent-termonology-menu-actions-dont.png)
+
+!!!
+
+When editing an object, use “Save” for the action on the page. When applying another action, use an imperative, present tense verb.
+
+!!! Do
+
+E.g. Edit organization. Action: Save organization
+
+![A modal that has a form to edit an organization](/assets/content/writing-style/writing-style-consistent-termonology-CTA-save-do.png)
+
+E.g. Invite users. Action: Invite users.
+
+![A form to invite users by inputting their emails](/assets/content/writing-style/writing-style-consistent-termonology-CTA-save-do.png)
+
+!!!
+
+!!! Dont
+
+E.g. Edit service principal. Action: Apply changes 
+
+![A form to edit a service principal with the wrong label in the primary button](/assets/content/writing-style/writing-style-consistent-termonology-CTA-apply-dont.png)
+
+!!!
+
+## Punctuation
+
+### Serial/Oxford comma
+
+Use the serial (“Oxford”) comma in lists.
+
+!!! Do
+
+Choose your cluster tier, size, and network.
+
+!!!
+
+!!! Dont
+
+Choose your cluster tier, size and network.
+
+!!!
+
+### Exclamation points
+
+Avoid using exclamation points. Only use it if you’re connecting with understood, strong reader emotion. 
+
+!!! Do
+
+Achieving this centralization is a huge improvement in security posture, but it’s not the end of the journey. This is because applications don't keep secrets! It turns out, most applications do a worse job keeping secrets than our close friends.
+
+!!!
+
+!!! Dont
+
+We are excited to announce Vault 1.3!
+
+!!!
+
+### Percentages
+
+Use the % symbol instead of spelling out "percent."  Correct: 50%
```

Comment:
This seems a bit more in line with how we usually write these types of statements.

```suggestion
Use the % symbol instead of spelling out "percent"; e.g. , `50%`.
```